src/widgets/ListNftWidget/subcomponents/ConnectedListNftWidget/ConnectedListNftWidget.tsx

```diff
@@ -135,25 +142,25 @@ const ConnectedListNftWidget: FC<ListNftWidgetProps> = ({
     setSelectedTokenId(value);
   };
 
+  useEffect(() => {
+    if (approveTransaction?.status === 'processing') {
```

Comment:
enum should be used for those kind of constant I think

Reply:
I agree. I will fix this in the #88 branch where I do a refactor on the transactions.